ui/app/styles/charts/colors.scss

```diff
@@ -114,6 +118,10 @@ $canceled: $dark;
     background: $lost;
   }
 
+  &.unknown {
+    background: $unknown;
+  }
+
```

Comment:
**question:** Should we also have a `disconnected` swatch and variable?

In `clients/:id`, I'm seeing this, and I think Nomad changes the status of the node to `disconnected` instead of `down` which we're not handling here:

![image](https://user-images.githubusercontent.com/41024828/163845670-3fc624f4-ea39-4e88-bc4a-8a3abc04d47e.png)

Reply:
Thanks; swatch added in a few more places:

![image](https://user-images.githubusercontent.com/713991/164368551-cd03d95f-29a0-4a56-a412-4c780ee28c76.png)
![image](https://user-images.githubusercontent.com/713991/164368568-211a78fd-869d-494f-ba73-89fdab2db093.png)
![image](https://user-images.githubusercontent.com/713991/164368578-34a217f1-7c3a-4c04-b096-5a2ba7d33ea2.png)